bench.go

```diff
@@ -17,6 +17,17 @@ import (
 	"time"
 )
 
+type nopWriter string
+
+func (wc nopWriter) Write(data []byte) (int, error) {
+	return len(data), nil
+}
```

Comment:
small nit: probably more standard to use io.Discard as a null writer instead of writing your own.